webxrlayers-1.bs

```diff
@@ -261,6 +261,14 @@ of either a {{WebGLRenderingContext}} or a {{WebGL2RenderingContext}}.
 Each {{XRLayer}} has an associated <dfn for="XRLayer">session</dfn>, which is the
 {{XRSession}} it was created with.
 
+<div class="algorithm" data-algorithm="setting the space on a layer">
```

Comment:
Should the space field just be on `XRLayer` and consolidate? we can use `partial interface` here.

Reply:
Equirect and projection layers unfortunately don't have a `space` attribute.

Reply:
Ah, I missed that. For some reason I thought projection layers aren't `XRLayer`s, but it's that they don't use <code>XLayer<i>Init</i></code>. 

We could _potentially_ have an `XRPlaceableLayer` interface that sits in between, but I'm not convinced that's super useful.

Reply:
Can we define it in IDL so it's just a helper class (ie you can't construct it)

Reply:
You can't construct IDL interfaces by default (this is true of XRLayer too)

Reply:
I'm mostly proposing this because I think it will simplify the spec text, but I'm not sure (and i'm not terribly attached to it).

Reply:
o yes, but there's still `instanceof`

Reply:
There's NoInterfaceObject, but we shouldn't do that. `instanceof` handles inheritance just fine.

The reason behind proposing this was not to make it easier to use from JS (it changes nothing), just to make it easier to consolidate spec text.